x/group/types_test.go

```diff
@@ -0,0 +1,125 @@
+package group_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/x/group"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestAllow(t *testing.T) {
```

Comment:
```suggestion
func Test PercentageDecisionPolicyAllow(t *testing.T) {
```

I guess we could do the same for the `ThresholdDecisionPolicy`